linux/kernel/building.md

```diff
@@ -278,4 +310,11 @@ kernel=kernel-myconfig.img
 
 This has the advantage of keeping your kernel separate from the kernel image managed by the system and any automatic update tools, and allowing you to easily revert to a stock kernel in the event that your kernel cannot boot.
 
+#### Extra for using a 64-bit kernel on Raspberry Pi OS (32-bit)
```

Comment:
I'm not sure if the kernel building page is the "right place" for this? Or if it is, there probably ought to be more explanation of how / why you'd want to use a 64-bit kernel with Raspberry Pi OS (32-bit) ?